src/lib/stepper/stepper-vertical.html

```diff
@@ -0,0 +1,37 @@
+<div class="mat-stepper-container">
+    <div *ngFor="let step of _steps; let i = index" class="mat-step" role="tablist">
+        <div #stepHeader class="mat-step-header" role="tab"
+             [id]="_getStepLabelId(i)"
+             [attr.aria-controls]="_getStepContentId(i)"
+             [attr.aria-selected]="selectedIndex == i"
+             [tabIndex]="_focusIndex == i ? 0 : -1"
+             (click)="select(step)"
+             (keydown)="_onKeydown($event)">
+            <div *ngIf="step.active" class="active-step">{{i + 1}}</div>
+            <div *ngIf="!step.active" class="inactive-step">{{i + 1}}</div>
+
+            <div class="mat-step-label">
+                <!-- If there is a label template, use it. -->
+                <ng-template [ngIf]="step.stepLabel">
+                    <ng-template [cdkPortalHost]="step.stepLabel"></ng-template>
+                </ng-template>
+                <!-- It there is no label template, fall back to the text label. -->
+                <ng-template [ngIf]="!step.stepLabel">
+                    <div>{{step.label}}</div>
+                </ng-template>
+            </div>
+
+        </div>
+        <div *ngIf="!step._isLast" class="connector-line"></div>
+        <div *ngIf="i == selectedIndex" role="tabpanel"
+             [id]="_getStepContentId(i)"
+             [attr.aria-labelledby]="_getStepLabelId(i)"
+             [attr.aria-expanded]="selectedIndex == i">
+            <ng-container [ngTemplateOutlet]="step.content"></ng-container>
+        </div>
+        <div *ngIf="i == selectedIndex">
+            <button md-button (click)="previous()">Back</button>
+            <button md-button (click)="next()">Next</button>
+        </div>
```

Comment:
Given that this div and the one on line 26 both have this condition, could you collapse them into one outer div (or ng-container) with the ngIf condition?